components/input/docs/partials/api.md

```diff
@@ -557,6 +557,51 @@ Use the `type="year-month-day"` attribute for a date formatted input.
 
 </auro-accordion>
 
+Use the `type="month"` attribute for a date formatted input.
```

Comment:
Looks like the examples got added but they don't appear on the page. When running a build, the `demo/api.md` file should have been updated as well. Looks like there is an issue with the building of the docs?

Reply:
known issue. will work on https://github.com/AlaskaAirlines/auro-form/issues/30